packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -712,12 +832,6 @@ async function handler(req: NextApiRequest & { userId?: number | undefined }) {
     return booking;
   }
 
-  if (reqBody.customInputs.length > 0) {
```

Comment:
Moved to `getCustomInputsResponses` which handles backward compatibility as well.